setup.py

```diff
@@ -44,6 +44,12 @@
     'description': "Build farm used to build the ROS ecosystem's packages.",
     'long_description': long_description,
     'license': 'Apache 2.0',
+    'entry_points': {
+        'colcon_core.verb': [
```

Comment:
I appreciate ros-buildfarm being in the verb name. I still find it confusing that the ros_buildfarm library installs colcon verbs but I may just have to get over that. I'm very much in favor of these verbs moving into the colcon org and being required by the jobs that need them.

Reply:
Hopefully @dirk-thomas can lend his thoughts on if/where these verbs could land in upstream colcon. For the record, I agree, and I hope we can pull these out.

Reply:
I commented on the two verbs inline. I am not really convinced they are needed (and if one of them is that it should be in this repo).

Reply:
The verbs have been dropped.